spec/features/issue_trash_recover_spec.rb

```diff
@@ -15,6 +15,8 @@
     visit "/trash"
     click_link 'Recover'
 
+    sleep(1)
```

Comment:
@etdsoft I haven't confirmed this yet, but I think the SQL query in `RecoverableRevisions` fails to order the issues if they are made within the same second. One solution is to refactor that query. I'm not sure why this test fails on this unrelated branch, though.